app/assets/javascripts/components/general_button.es6.jsx

```diff
@@ -25,14 +30,23 @@ class GeneralButton extends React.Component {
         padding: '6px 12px',
         color: 'white',
         backgroundColor: '#68B1DE',
-        borderBottom: '2px solid',
-        borderColor: '#28719E',
+        borderBottom: '2px solid #28719E',
         borderRadius: 2,
         fontSize: '14px',
       },
+      hover: {
+        borderBottom: 0,
+      },
     };
   }
 
+  componentDidMount() {
```

Comment:
We will most likely be using `componentWillReceiveProps` in some components, but it's not applicable in this component just yet!